controllers/v1/observationsController.js

```diff
@@ -982,42 +993,67 @@ module.exports = class Observations extends Abstract {
                     result: {}
                 };
 
+                //eG change
                 let observationDocument = await database.models.observations.findOne({ 
```

Comment:
@VISHNUDAS-tunerlabs use observationDocument helper to query data. remove directly querying from db